examples/TripPin.openapi3.json

```diff
@@ -4165,6 +4219,9 @@
                 "title": "Event (for update)",
                 "type": "object",
                 "properties": {
+                    "@odata.type": {
+                        "$ref": "#/components/schemas/type"
+                    },
                     "ConfirmationCode": {
                         "type": "string",
                         "nullable": true
```

Comment:
@ralfhandl I'm having an odd issue with the test claiming stuff isn't there when it is

```
1) Examples
       TripPin:

      AssertionError [ERR_ASSERTION]: OpenAPI document
      + expected - actual

               "type": "object"
             }
             "Microsoft.OData.SampleService.Models.TripPin.Event-update": {
               "properties": {
      +          "@odata.type": {
      +            "$ref": "#/components/schemas/type"
      +          }
                 "ConfirmationCode": {
                   "nullable": true
                   "type": "string"
                 }
```
as you can see in the diff, it's quite clearly there. No idea what going on :(

Reply:
The unit test `npm run test` compares the current in-memory output with the "expected" output from the `examples` folder.

If you are satisified with the diff, you can run `npm run build` to refresh the "expected" output. Use with care 😄

Reply:
part of the problem here was the `+ expected - actual` output not having enough context to identify where the issue was.

I ended up running `npm test > report` and counting how many lines it wrote to identify if I removed or added the `@odata.type` in the correct.

It wasn't fun :(